driver/pace/driver/report.py

```diff
@@ -37,15 +37,20 @@ def __post_init__(self):
 
 
 def get_experiment_info(
-    experiment_name: str, time_step: int, backend: str, git_hash: str
+    experiment_name: str,
+    time_step: int,
+    backend: str,
+    git_hash: str,
+    is_orchestrated: bool,
 ) -> Experiment:
+    orchestration = "orchestrated" if is_orchestrated else "python"
     experiment = Experiment(
         dataset=experiment_name,
         format_version=3,
         git_hash=git_hash,
         timestamp=datetime.now().strftime("%d/%m/%Y %H:%M:%S"),
         timesteps=time_step,
-        backend="python/" + backend,
+        backend=f"{orchestration}/{backend}",
```

Comment:
This is a positive change, but `backend` is a path string and is overloaded as a gt4py backend, dace backend, etc. Can that be disambiguated from these? Perhaps `backend_path`?

Reply:
Consider also using https://docs.python.org/3/library/pathlib.html.

Reply:
Not sure I understand. Backend is the string repr of the backend name here

Reply:
ah ok, so it's not actually a path on disk. that was a misunderstanding on my part. all good!